src/chrome/content/rules/KickassTorrents.xml

```diff
@@ -1,20 +1,18 @@
-<!--
-	Requested: https://trac.torproject.org/projects/tor/ticket/9232
-	Updated: https://trac.torproject.org/projects/tor/ticket/10232
-
--->
 <ruleset name="KickassTorrents">
-  <target host="kat.ph" />
-  <target host="www.kat.ph" />
-  <target host="kickass.to" />
-  <target host="www.kickass.to" />
-  <target host="kat.cr" />
-  <target host="www.kat.cr" />
-
-  <rule from="^http://(?:www\.)?k(?:at\.ph|ickass\.to)/"
-	  to="https://kat.cr/" />
+	<target host="kat.ph" />
+	<target host="www.kat.ph" />
```

Comment:
for that hosts exist some language specific sub domains, but I think that's okay since they seems to be deprecated anyway.